doc/source/rest.j2

```diff
@@ -87,18 +101,45 @@ status code. It is possible to provide any number of |measures|.
 
 .. IMPORTANT::
 
-   While it is possible to send any number of (timestamp, value), it is still
-   needed to honor constraints defined by the |archive policy| used by the
+   While it is possible to send any number of (timestamp, value), they still
+   need to honor constraints defined by the |archive policy| used by the
    |metric|, such as the maximum |timespan|.
 
+Batch
+~~~~~
+
+It is also possible to batch |measures| sending, i.e. send several |measures|
+for different |metrics| in a simple call:
+
+{{ scenarios['post-measures-batch']['doc'] }}
+
+Or using named |metrics| of |resources|:
+
+{{ scenarios['post-measures-batch-named']['doc'] }}
+
+If some named |metrics| specified in the batch request do not exist, Gnocchi
+can try to create them as long as an |archive policy| rule matches:
+
+{{ scenarios['post-measures-batch-named-create']['doc'] }}
+
```

Comment:
Since that measures and aggregates have been distinguished now, maybe there should be a new section:
```
Aggregates
==========
```